frontend/src/components/layouts/AppLayout.tsx

```diff
@@ -242,7 +242,7 @@ export default function AppLayout() {
             </div>
           </div>
           <main className="flex flex-col">
-            <header className="md:hidden flex h-14 items-center gap-4 border-b bg-muted/40 px-4 lg:h-[60px] lg:px-6 justify-between">
+            <header className="md:hidden sticky top-0 z-50 flex h-14 items-center gap-4 border-b bg-muted/40 backdrop-blur px-4 lg:h-[60px] lg:px-6 justify-between">
```

Comment:
I made the navbar sticky because it felt annoying to scroll up everytime to access the sidebar, let me know if it's okay